.github/workflows/release-layer-python.yml

```diff
@@ -0,0 +1,57 @@
+name: "Release Python Lambda Layer"
+
+on:
+  # (Using tag push instead of release to allow filtering by tag prefix.)
+  push:
+    tags:
+      - layer-pythonjs/**
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build-layer:
+    runs-on: ubuntu-latest
+    outputs:
+      PYTHON_SDK_VERSION: ${{ steps.save-python-sdk-version.outputs.PYTHON_SDK_VERSION}}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 14
+
+      - name: Build
+        run: |
+          PYTHON_SDK_VERSION=$(bash run.sh -n opentelemetry-python-layer -b true 2>&1 > /dev/null | sed -n "s/opentelemetry-sdk==\(.*\)/\1/p")
+        working-directory: python/sample-apps
+
+      - name: Save Python SDK Version
+        id: save-python-sdk-version
+        run: |
+          echo "PYTHON_SDK_VERSION=$PYTHON_SDK_VERSION" >> $GITHUB_OUTPUT
+        working-directory: python/packages/layer/scripts
+
+      - uses: actions/upload-artifact@v3
+        name: Save assembled layer to build
+        with:
+          name: opentelemetry-python-layer.zip
+          path: python/packages/layer/build/opentelemetry-python-layer.zip
+
+  publish-layer:
+    uses: ./.github/workflows/layer-publish.yml
+    needs: build-layer
+    strategy:
+      matrix:
+        aws_region: 
+          - us-west-1
+          - us-west-2
```

Comment:
probably want the other regions listed out in comments as a reminder for future official release.

Reply:
Fixed